### ISSUE TITLE
Wrap "run cell" in a method?

### ISSUE THREAD

**Issue by ma-sadeghi** [hoangKnLai/vscode-ipython#19]
Currently, if I run a cell, the entire contents of the cell, which might be tens of lines of code gets pasted in the console. It'd be nice if you could wrap the call in a method like `runcell(cell_id, file)`, which would show up instead in the console. This is how it's done in Spyder and I really like it as it keeps the console clean. Here's a screenshot from Spyder:

![image](https://user-images.githubusercontent.com/14086031/190051913-e58607f9-a780-433a-b0a3-771f245c5f33.png)

**Comment by hoangKnLai** [comment]
TL;DR:  to do what you're asking I think there needs to be a IPython API in typescript and/or javascript before an attempt to integrate it with VSCode.  The API piece likely can only be done well by the IPython team.  

The long version: I would like this too and tried to do something similar with the `%load` command in the previous major release.  The `%load` command tends to have issue with autoindentation and empty newline between code blocks.  Hence, the motivation toward the recent changes to self handle the code loading to the terminal instead of `%load`. 

I also did look into IPython REPL and how Spyder uses it and here are what I found:  
- The IPython API is in Python only when I last checked (no typescript / javascript for use with VSCode)
- Spyder uses the IPython API by
    - Take the python code snippet as text (sometime written out to a hidden file to format the text)
    - Tell Python to compile the code snippet
    - Execute the compiled code using Python `exec` with the current `IPython` workspace as the Python variable workspace

VSCode is typescript and I'm no typescript expert.  All I found and attempted so far: 
- Execute Python as a sub-routine to startup a QT IPython version (same as IPython but as a separate QT GUI)  
- Use the Jupyter notebook like comlink over TCP to send code and get results from the IPython QT terminal
    - For some reason, the IPython QT terminal do execute the command but not rendering any of the input / output on terminal

**Comment by hoangKnLai** [comment]
Updated to include a `Send Command` option that uses `file` instead of `clipboard`.  The `file` is similar in which is shows a dozen lines or so of the bottom of the code block and nothing more.  See if this is ok enough to close this issue?
